src/util/diproche/diprocheInterface.ts

```diff
@@ -0,0 +1,6 @@
+// Please specify and describe the return type for this function.
+// Input should be a string I can just run in diproche
+// (also the corresponding predicate like diproche_fo(...).)
+export default function getErrors(diprocheInput: string) {
```

Comment:
What output would we expect from this function at all? All it can do is find issues in the proof provided to it. IMHO it can just add these issues to the global issue-list itself and return `void`.